oauth_client.go

```diff
@@ -82,10 +82,17 @@ type OAuthClient struct {
 	OAuthTokens  []*OAuthToken `jsonapi:"relation,oauth-tokens"`
 }
 
+// A list of relations to include
+type OAuthClientIncludeOp string
+
+const OauthClientOauthTokens OAuthClientIncludeOp = "oauth_tokens"
```

Comment:
Besides "OAuthToken" another relation listed for `type OAuthClient struct` is "organization". However,  I did not included a constant for "organization" because I did not see it listed in atlas's `allowed_values_for_include`